### PR TITLE
test: fix master CI fail

### PR DESCRIPTION
**Pull request by li-jia-nan**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | test: fix master CI fail |
| 🇨🇳 Chinese | test: fix master CI fail |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by li-jia-nan** [comment]
这里不知道为啥会挂，先把 CI 修了，具体原因还需要排查一下，cc @MadCcc @zombieJ

File: components/theme/__tests__/token.test.tsx [hunk 1]
```diff
@@ -326,18 +326,18 @@ describe('Theme', () => {
...
-      'background-color': '#6ce0a4',
+    expect(container.querySelector<HTMLElement>('.ant-slider-track')).toHaveStyle({
+      'background-color': '#44D48E',
     });
 
     rerender(<Demo algorithm={theme.darkAlgorithm} />);
-    expect(container.querySelector('.ant-slider-track')).toHaveStyle({
-      'background-color': '#0e462e',
+    expect(container.querySelector<HTMLElement>('.ant-slider-track')).toHaveStyle({
+      'background-color': '#0B5E3B',
```
[the remark this answers is not in the document]
Comment (yoyo837) [reply components/theme/__tests__/token.test.tsx:340]: 小弟试试找下为什么

[the remark this answers is not in the document]
Comment (afc163) [reply components/theme/__tests__/token.test.tsx:340]: https://github.com/ant-design/ant-design/pull/48571 这里改的。

[the remark this answers is not in the document]
Comment (yoyo837) [reply components/theme/__tests__/token.test.tsx:340]: 他那个PR怎么CI不报错

[the remark this answers is not in the document]
Comment (yoyo837) [reply components/theme/__tests__/token.test.tsx:340]: 看到豆酱的PR了，jsdom有关

**Pull request closed without merging**